udata/migrations/2021-07-07-update-schema-for-community-resources.py

```diff
@@ -0,0 +1,31 @@
+'''
+The purpose here is to update every community resource's metadata 'schema'
+with a new format (string to object)
+'''
+import logging
+
+from udata.models import CommunityResource
+
+log = logging.getLogger(__name__)
+
+
+def migrate(db):
+    log.info('Processing community resources.')
+
+    community_resources = CommunityResource.objects().no_cache().timeout(False)
+    for resource in community_resources:
+        save_res = False
+        if hasattr(resource, 'schema'):
+            schema = resource.schema
+            resource.schema = {}
+            if schema is not None and isinstance(schema, str):
```

Comment:
Could schema be another type? Maybe log a warning if it is the case?

Reply:
The community resource model had a stringfield before and a dictfield now. So the schema cannot be another type than str